src/core-services/account/schemas/account.graphql

```diff
@@ -136,6 +136,18 @@ input RemoveAccountEmailRecordInput {
   email: Email!
 }
 
+"Defines groups to be removed from an account group. Removin an group from an account effectively also removes permissions from the account"
+input RemoveUserPermissionsInput {
+  "the opaque account Id of the user who is to be removed from the group"
+  accountId: String!
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+  "the groups to remove from a user / account"
+  groups: [String!]!
```

Comment:
This should be `groups: [ID]!` as groups are stored by their ID in the database on an account. Then you decode the array of ids in the resolver and pass them to the mutation.